bin/add-term

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const fs = require('fs-extra')
+
+const processTerm = (term) => {
```

Comment:
can you rename this method to what it does. `capitalizeTerm`? Also are you sure all terms should be capitalized? I'm not sure we want this restriction

Reply:
if we do end up needing this, we could write it like this:
```js
const capitalizeTerm = term =>
  term.split(' ').map(term => term[0].toUpperCase() + term.slice(1)).join(' ')
```